### ISSUE TITLE
Release: July 15, 2026

### ISSUE THREAD

**Issue by baptisten** [JudicialCouncilOfCalifornia/trialcourt#5993]
https://caljc.sharepoint.com/:x:/r/sites/WebServices2/Shared%20Documents/Testing/Smoke%20Testing%20Checklist.xlsx?d=wfa4d821a7c0d483582ff0f83fc46305a&csf=1&web=1&e=CFRaeO&nav=MTVfezBGRjgxNEQ1LTlCRjUtNDYxQy1BMjdBLUQ5ODFFMDhCOTFCNn0

**Comment by baptisten** [comment]
## SRL
[feature/ADA-367-tooltips](https://github.com/JudicialCouncilOfCalifornia/jcc-srl/tree/feature/ADA-367-tooltips)
- Tooltips are not announced to the screenreader when focused

**Comment by alexwu-jcc** [comment]
**TrialCourt**

[bugfix/ada-482](https://github.com/JudicialCouncilOfCalifornia/trialcourt/tree/bugfix/ada-482)

- Supreme Court - Color Contrast issue for login page


[bugfix/tci_1398__i](https://github.com/JudicialCouncilOfCalifornia/trialcourt/tree/bugfix/tci_1398__i)

- Update the defaults for the case category so cases will be rendered by regions

REMOVING FROM DEPLOYMENT, July 15, 2026
~~[bugfix/ada-240--ada-264](https://github.com/JudicialCouncilOfCalifornia/trialcourt/tree/bugfix/ada-240--ada-264)~~

~~- Let the last element in breadcrum to be emphased.~~


[bugfix/ada-240--ada-276](https://github.com/JudicialCouncilOfCalifornia/trialcourt/tree/bugfix/ada-240--ada-276)

- Alameda.courts.ca.gov - Footer

MOVING TO NEXT DEPLOYMENT
~~[bugfix/ada-240--ada-302](https://github.com/JudicialCouncilOfCalifornia/trialcourt/tree/bugfix/ada-240--ada-302)~~

~~- Changed the "Expand All" and "Close All" from anchor to button, etc.~~

ALREADY MERGED
~~[bugfixing/ada-240-and-ada-278](https://github.com/JudicialCouncilOfCalifornia/trialcourt/tree/bugfixing/ada-240-and-ada-278)~~

~~- Added add H5 option to aside so use can go to /self-help and change the font of aside from h4 to h5~~


---

**Terminus**

[bugfix/tci_1398](https://github.com/JudicialCouncilOfCalifornia/terminus-actions/tree/bugfix/tci_1398)

- Changed the cron job schedule for San Mateo
